### PR TITLE
build: button component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

**Review by Omkar-Ghate — CHANGES_REQUESTED** [review]
You can change the cursor to a pointer after hovering over the buttons. Also, the two top right buttons in the navigation are overlapped, do fix them.

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
+
+body{
+    background-image: none;
+}
+
```
Comment (developer-anand) [inline components/button/button.css:7]: remove extra space.

**Review by developer-anand — APPROVED** [review]
All good to merge

**Review by arjun-computer-geek — APPROVED** [review]
Looking good to me. You can merge it

**Review by Omkar-Ghate — APPROVED** [review]

**Pull request merged**